mockingbird/src/jvmTest/kotlin/com/careem/mockingbird/test/FreezeStateVerification.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Careem, an Uber Technologies Inc. company
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.careem.mockingbird.test
+
+import kotlin.test.assertFalse
+
+actual fun <T> T.verifyFreezeStateForMultiThread() {
+    assertFalse { this.isFrozen }
+}
+
+actual fun <T> T.verifyFreezeStateForLocalThread() {
+    assertFalse { this.isFrozen }
+}
```

Comment:
I think this can be empty implementation, since isFrozen doesn't take effect on js or jam